styles.css

```diff
@@ -0,0 +1,250 @@
+* {
+    padding:0;
+    margin:0;
+    
+}
+body {
+}
+header {
+    width: 100%;
+    height:225px;
+    background-color:navy;
+
+}
+.header-holder {
+    width:90%;
+    margin: 0px auto;
+    display:flex;
+    justify-content: space-between;
+}
+
+.left-side-header {
+   margin-top: 3%;
+   color:lightgreen;
+}
+.left-side-header img {
+    width:75%;
+}
+.hidden-header {
+    font-size: 1px;
+    visibility:hidden;
+}
+.left-side-header p {
+    font-size:1.5em;
+    font-style:italic;
+}
+.right-side-header {
+    color:white;
+    margin-top: 3%;
+}
+.shopping-cart {
+    width:20px;
+    padding-right: 2px;
+}
+.right-side-header button{
+    margin-top: 3%;
+    width:100%;
+}
+nav {
+    width:100%;
+    height:80px;
+    background-color:beige;
+
+}
+ul.header-nav-list-holder {
+    margin-left:8%;
+    height:80px;
+    line-height:80px;
+    margin-right:5px;
+    display:flex;
+    flex-direction:row;
+    list-style-type: none;
+}
+a {
+    color:navy !important;
+    text-decoration:none;
+}
+a.needs-slash::after {
+    content: "/";
+    padding-left:10px;
+    margin-right:-5px;
+}
+a.needs-slash2::after {
+    content: "/";
+    padding-left:5px;
+    padding-right: 5px;
+}
+li.header-nav-list-item {
+    
+    margin-right:20px;
+}
+.dogs-list-header {
+    margin-left: 7.5%;
+    margin-top: 2%;
+}
+.page-header {
+    margin-left: 7.5%;
+    margin-top:2%;
+    margin-bottom:2%;
+}
+/*
+
+#wrapper {
+    width:85%;
+    height:1000px;
+    margin:0px auto;
+    background-color:white;
+    
+}
+*/
+.grid-item-container {
+    width:85%;
+    margin: 0px auto;
+    display:grid;
+    column-gap: 6.67%;
+    grid-template-columns: 20% 20% 20% 20%;
+    grid-template-rows: auto;
+ 
+}
+.initial-box {
+    margin-top:20px;
+    grid-column: 1/ span 4;
+    display:flex;
+    flex-direction:row;
+    
+}
+.sub-initial-box {
+    margin-left: 6.67%;
+}
+h2.adopt-primary-header {
+    margin-bottom: 10px;
+}
+.intro-para {
+    line-height:35px;
+}
+.adoption-pic {
+    width:100%;
+}
+.initial-box img {
+    width:30%;
+}
+.dog-holder {
+    margin-top: 20%;
+    border-style: outset;
+    padding:5px;
+}
+dog-cost {
+    font-weight:600;
+}
+.adopt-button {
+  width:100%;
+  height:30px;
+  color:white;
+  background-color:green;
+}
+footer {
+    margin-top:50px;
+    height:350px;
+    background:darkgray;
+    clear:both;
+    display:flex;
+    flex-flow:row wrap;
+    justify-content:center;
+    align-items:center;
+}
+footer img {
+    width:100%;
+    display:block;
+}
+.blog-main-holder {
```

Comment:
You have a grid property around your blog posts, that isn't really necessary.

Reply:
I used to grid properties to adjust the positions of things. Should I figure out an alternative?

Reply:
It's just overkill. Use grid for big grid things, and most small components can be handled with some of the more basic things. I'd argue you could lay out these articles using just flexbox. Maybe even just simple display properties. 

Also `.grid-item-container` isn't really doing much with the grid. I turned off all the properties in the inspector and nothing changed.